docs/tutorial/updates.md

```diff
@@ -94,3 +96,6 @@ autoUpdater.on('error', message => {
   console.error(message)
 })
 ```
+
+[electron-builder-lib]: https://github.com/electron-userland/electron-builder
+[electron-updater]: https://github.com/electron-userland/electron-builder/wiki/Auto-Update
```

Comment:
![image](https://user-images.githubusercontent.com/5731176/30336412-8757bf98-97bb-11e7-8443-5e527542459e.png)

Use direct redirection to https://www.electron.build/auto-update